### PR TITLE
POST /api/v1/playlists

### PR DESCRIPTION
**Pull request by nathangthomas**
This PR does the following: 
- Updates app.js file to include playlists route
- Adds spec for `POST /api/v1/playlists`
- Adds `POST /api/v1/playlists` endpoint

### CODE REVIEW

File: routes/api/v1/playlists.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  var body = request.body;
+  var title = body.title;
+
+  for (let requiredParam of ['title']) {
+    if (!body[requiredParam]){
+      return response
+        .status(400)
+        .send({ error: `Missing required attribute <${requiredParam}>` });
+    }
+  }
```
Comment (fentontaylor) [inline routes/api/v1/playlists.js]: Seems a little overkill if there is only one attribute to check. How about replacing with:
```
if (!title) return response.status(400).send({ error: `Missing required attribute <title>` });
```

File: tests/playlists/create.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    await database.raw("TRUNCATE TABLE playlists CASCADE");
+  });
+
+  it("it can create a playlist", async () => {
+    const body = {
+      "title": "Twistin' Grips"
+    };
+
+    const noFavs = await database('playlists').first()
+    expect(noFavs).toBeUndefined();
```
Comment (fentontaylor) [inline tests/playlists/create.spec.js]: Should update this variable from `noFavs` to `playlist`.

File: routes/api/v1/playlists.js (hunk 1 above)
Comment (nathangthomas) [reply routes/api/v1/playlists.js]: Yeah, it's a bit much.

File: tests/playlists/create.spec.js (hunk 2 above)
Comment (nathangthomas) [reply tests/playlists/create.spec.js]: Good catch. Should be more careful with copy and pasting from other test files.

**Pull request merged**